pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -103,7 +103,9 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
         super.channelActive(ctx);
 
         if (proxyToTargetBrokerAddress == null) {
-            log.info("{} Connected to broker", ctx.channel());
+            if (log.isDebugEnabled()) {
```

Comment:
If it is not creating lot of noise then should we keep it as info for any future troubleshooting, it will give us idea that at least client is connected to broker.?

Reply:
@rdhabalia the line I've pointed above (https://github.com/apache/incubator-pulsar/blob/master/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConnectionPool.java#L157) is where we would still have the info log message when the client gets connected to broker